src/xtgeo/grid3d/grid_properties.py

```diff
@@ -410,23 +410,21 @@ def from_file(
 
         pfile.check_file(raiseerror=OSError)
 
+        properties = []
```

Comment:
Do we need to initialise an empty list here since we raise OSError in `else`?

Reply:
True, fixed!